abci/abci_test.go

```diff
@@ -1099,7 +1098,7 @@ func (s *ProposalsTestSuite) TestProcessProposal() {
 		s.Require().NoError(err)
 
 		// Mev lane
-		mevLane := s.setUpTOBLane(math.LegacyMustNewDecFromStr("0.0"), map[sdk.Tx]bool{})
+		mevLane := s.setUpTOBLane(math.LegacyMustNewDecFromStr("0.3"), map[sdk.Tx]bool{})
```

Comment:
Multiple lanes were using max block space of 0 (indicating "unlimited") so this needed to be changed throughout.